app/views/companies/show.html.haml

```diff
@@ -29,15 +29,18 @@
           %h2 #{t('.address')}:
           = field_or_none t('.street'), @company.main_address.street_address, tag_options: {class: 'street'}
 
+          = field_or_none t('.post_code'), @company.main_address.post_code, tag_options: {class: 'post-code'}
+
           = field_or_none t('.city'), @company.main_address.city, tag_options: {class: 'city'}
 
           = field_or_none t('.kommun'), @company.main_address.kommun.name, tag_options: {class: 'kommun'}
```

Comment:
@weedySeaDragon - yes, I meant that this statement was failing - for some companies - when I ran in my local environment.  This is because some records did not have a `kommun` associated with them.  This caused a ```NoMethodError``` when calling `name` on `nil`.

The `&.` is the "safe navigation operator" (https://makandracards.com/makandra/41208-ruby-2-3-0-has-a-safe-navigation-operator).  It may not be needed if we are sure we will _always_ have a kommun, but may be a good "belts and suspenders" move.